src/Symfony/Component/Serializer/Annotation/SerializedName.php

```diff
@@ -28,6 +28,11 @@ final class SerializedName
      */
     private $serializedName;
 
+    /**
+     * @var array
+     */
```

Comment:
I think the phpdoc can be removed as the default value already hints the correct type.

Reply:
done